src/main.rs

```diff
@@ -126,18 +127,23 @@ struct Args {
     #[clap(long, default_value_t = 15 * 60)]
     timeout_starred: u64,
 
-    /// The lowest listen port to be allocated via docker.
+    /// The lowest listen port to be allocated via the selected OCI container engine.
     #[clap(long, default_value_t = 0)]
     port_min: u16,
 
-    /// The highest listen port to be allocated via docker.
+    /// The highest listen port to be allocated via the selected OCI container engine.
     #[clap(long, default_value_t = 0)]
     port_max: u16,
 
     /// The maximum number of listen ports a workload is allowed to have (0 to disable).
     #[clap(long, default_value_t = 0)]
     listen_max: u16,
 
+    /// OCI container engine command to execute, for example, `docker` or `podman`.
+    /// This may also be an absolute path.
+    #[clap(long, default_value = "docker")]
+    command: OsString,
```

Comment:
Maybe call this `oci_runtime_command`, so that people can't accidentally get `command` set from an old configuration, and have things blow up?

Reply:
We're the only users of this, but I agree. I'll try to get the `oci` into the name of this new argument to address https://github.com/profianinc/benefice/pull/128#discussion_r951336538 as well